extensions/pom.xml

```diff
@@ -2122,18 +2109,6 @@
       <version>${annogen.version}</version>
       <scope>runtime</scope>
     </dependency>
-    <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
-      <artifactId>httpcore</artifactId>
-      <version>${httpcore.version}</version>
-      <scope>runtime</scope>
```

Comment:
Did you double check if the httpclient lib was still being included in the `lib`? It probably isn't it is probably needed...

Reply:
A pipeline was tested without this, and the lib was still being included.